.github/workflows/create-release-draft.yml

```diff
@@ -278,6 +278,8 @@ jobs:
       - name: Generate release notes
         run: |
           ./scripts/generate-release-notes.sh ${{ github.workspace }}/.github/release_notes.md ${{ needs.set-release-type.outputs.release_type }} ${{ env.DIFF_TAG }}
+        env:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Is this required?

Reply:
gh pr list should need it.

Reply:
tried below code:
![image](https://github.com/litentry/litentry-parachain/assets/120463031/a8c6f39f-de4f-424e-9707-3d971abc2978)

Result:

![image](https://github.com/litentry/litentry-parachain/assets/120463031/f1cd52e5-acbd-48e3-87e7-83a8555a1c6f)

Reply:
Yes. There is `secrets.GITHUB_TOKEN`, but not `env.GH_TOKEN`. Where do we need `env.GH_TOKEN`? If so, why not use `secrets.GITHUB_TOKEN` directly?

Reply:
gh pr list --search needs it

This is the suggested way from github, see the button part of the image: 

https://user-images.githubusercontent.com/120463031/250709186-69a435a0-b167-4a1f-b1e8-d3ad70afcbfb.png